lib/reline/line_editor.rb

```diff
@@ -2264,7 +2264,7 @@ def finish
   end
 
   private def vi_histedit(key)
-    path = Tempfile.open { |fp|
+    path = Tempfile.open(["reline", ".rb"]) { |fp|
```

Comment:
I couldn't find an obvious test to change for this. I'm happy to make such a change if necessary.

Also I wasn't sure about whether to add the `reline` to the file or some other static string. It seems that `Tempfile` requires me to put some file name in there if I want to add an extension.